### ISSUE TITLE
JetFormBuilder - Disable past date in datepicker & Datetime (Code example added)

### ISSUE THREAD

**Issue by Lonsdale201** [CrocoBlock/suggestions#5620]
Can you add in a future an option, for the Datepicker, and Datetime, to disable the past date? :)
https://prnt.sc/yyl4pVrm1qYT

At the moment you can do this with the following code: (just dont forget, when you add a date field in the jfb, you need to add the same ID in the code example also "date_picker" .  https://prnt.sc/VxRlkX_zHf_T
(This snippet work with the Date picker)

```
var today = new Date();
var dd = String(today.getDate()).padStart(2, '0');
var mm = String(today.getMonth() + 1).padStart(2, '0');
var yyyy = today.getFullYear();

today = yyyy + '-' + mm + '-' + dd;
 jQuery(document).ready(function ($) {
$('#date_picker').attr('min',today);
    });
```
    
    
    
    
If you need tha same functionality, but with the **datetimepicker** field:
(This snippet work with the Datetime picker)
```
jQuery(document).ready(function ($) {
  const today = new Date();
  const year = String(today.getFullYear()).padStart(4, '0');
  const month = String(today.getMonth() + 1).padStart(2, '0');
  const date = String(today.getDate()).padStart(2, '0');
  const hours = String(today.getHours()).padStart(2, '0');
  const minutes = String(today.getMinutes()).padStart(2, '0');
  const minDateTime = `${year}-${month}-${date}T${hours}:${minutes}`;
  $('#date_picker').attr('min', minDateTime);
});    
```
    
   We followed the same identifier as the previous one. Visually it will not let you select the date, the hour minute selector will set it to the current time, you can go back to an earlier time, but it will indicate when validating that it should be later than the current time. So when you press the submit button, even if you select an older time, it will not accept it when validating. This is only valid for the day that is currently "today"

**Comment by manashjyotix** [comment]
Where I put this code?

**Comment by Lonsdale201** [comment]
If you have elementor pro, you can use the custom code, function, or something else code snippet plugins which support js, or, you can use html widget / blokk to, just dont forget the <script></script> tag.

**Comment by moxet** [comment]
@Lonsdale201 i tried it on Jet Form (Jetengine Native) with no success. I can see the min attribute in HTML but the dates are still enable, any guess over JetForms?

**Comment by Lonsdale201** [comment]
> @Lonsdale201 i tried it on Jet Form (Jetengine Native) with no success. I can see the min attribute in HTML but the dates are still enable, any guess over JetForms?

This option only work with date picker.

https://prnt.sc/hepPzjcyDTKg

**Comment by Lonsdale201** [comment]
But it doesn't matter if you use the obsolete legacy version or the JetFormbuilder date selector field, the important thing is to enter the ID correctly, according to the script. In my example, you need to enter #date_picker in the "name" field. If you rewrite it, change the ID to the Form as well.

**Comment by moxet** [comment]
I've valid ID of the field because i can see in the HTML output the MIN is applied as today date. But i am using DateTime Picker not Date picker, it might be the issue?

**Comment by Lonsdale201** [comment]
> I've valid ID of the field because i can see in the HTML output the MIN is applied as today date. But i am using DateTime Picker not Date picker, it might be the issue?

Yes, and sorry its my fault, to not indicated, this solution only work with the date picker at the moment. I don't have much time now, but I'll look at DatePicker later. :)

**Comment by moxet** [comment]
Thank you, @Lonsdale201 highly appreciate

**Comment by Lonsdale201** [comment]
> Thank you, @Lonsdale201 highly appreciate

I'm a bit late, but I updated the code for datetimepicker. :)
